ai_research/dataset_generation/domain/llm_response.py

```
@@ -0,0 +1,26 @@
+        start = match.end()
+        end = matches[index + 1].start() if index + 1 < len(matches) else len(response)
+        sections.append(response[start:end].strip())
+    return sections
+
+
+def parse_examples(response: str) -> list[Example]:
+    examples: list[Example] = []
+    for section in parse_example_sections(response):
+        try:
+            examples.append(parse_example(section))
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)